ethportal-api/src/types/execution/block_body.rs

```diff
@@ -123,17 +123,16 @@ impl BlockBody {
     /// Returns reference to uncle headers.
     ///
     /// Returns None post Merge fork.
```

Comment:
```suggestion
    /// Returns None post Merge fork.
```

this doc is no longer true